aptos-move/framework/aptos-stdlib/sources/cryptography/ristretto255.move

```diff
@@ -167,10 +167,10 @@ module aptos_std::ristretto255 {
     }
 
 
-    /// Hashes the input to a uniformly-at-random RistrettoPoint via SHA512.
-    public fun new_point_from_sha512(sha512: vector<u8>): RistrettoPoint {
+    /// Hashes the input to a uniformly-at-random RistrettoPoint via SHA2-512.
+    public fun new_point_from_sha2_512(sha2_512_input: vector<u8>): RistrettoPoint {
```

Comment:
@davidiw, @wrwg and @movekevin, can I actually rename this function at this point?

Reply:
no, that's a breaking change.

Reply:
Can I change the argument names? e.g., `sha512` to `sha2_512`?